src/main/resources/META-INF/rewrite/java-ee-7.yml

```diff
@@ -25,6 +25,18 @@ recipeList:
   - org.openrewrite.java.migrate.javaee7.OpenJPAPersistenceProvider
   - org.openrewrite.java.migrate.JpaCacheProperties
   - org.openrewrite.java.migrate.BeansXmlNamespace
+  - org.openrewrite.java.migrate.javaee7.openJPA
+
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.javaee7.openJPA
+displayName: Migrate to JavaEE7 OpenJPA
+description: These recipes help with the Migration to Java EE 7 OpenJPA, flagging and updating deprecated methods.
+tags:
+  - javaee7
+  - deprecated
+recipeList:
+  - org.openrewrite.java.migrate.javax.AutoGeneratedValue
```

Comment:
The name of the recipe now reflects what's matched, but not what the intended change is. Perhaps we can arrive at a mre descriptive recipe name such that it's clearer to users? Something like `AddTableGenerator` already shows more of the intended change, but perhaps you have other ideas still

Reply:
We've been aiming to have the name consistent with the ID we use in our tool, but I understand some of the names don't always conform with the Open Rewrite convention. I'll go ahead and update the name to `AddTableGenerator`.